images/runtime/python/template.Dockerfile

```diff
@@ -13,6 +13,7 @@ ENV BUILD_NUMBER=${BUILD_NUMBER}
 RUN ./build.sh python /opt/startupcmdgen/startupcmdgen
 
 FROM oryxdevmcr.azurecr.io/private/oryx/%BASE_TAG% as main
+ARG DEBIAN_FLAVOR
 ARG IMAGES_DIR=/tmp/oryx/images
 ARG BUILD_DIR=/tmp/oryx/build
 ENV DEBIAN_FLAVOR=${DEBIAN_FLAVOR}
```

Comment:
Same here, I'm just a bit curious why we have debian argument at line 16 when we previously use line 19 to set up environment variable.

Reply:
This is because in the added line below, I could not find a way that the env var could be used in the COPY docker command. [It looked like](https://stackoverflow.com/a/55255444) I needed to place the debian flavor in the build context by making it an argument.